pubspec.yaml

```diff
@@ -30,6 +30,7 @@ dependencies:
   cupertino_icons: ^1.0.2
```

Comment:
remove flutter-session dependency as it is not used anywhere yet

Reply:
ok